lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -201,7 +206,8 @@ def self.go_template_for(kind, name)
           %[(eq .involvedObject.kind "#{kind}")],
           %[(eq .involvedObject.name "#{name}")],
           '(ne .reason "Started")',
-          '(ne .reason "Created")'
+          '(ne .reason "Created")',
+          '(ne .reason "SuccessfulCreate")'
```

Comment:
`Scheduled` is often useful in determining progress when new nodes have to be allocated, I would keep the messages.